### PR TITLE
qm-monitor: add page

### PR DESCRIPTION
**Pull request by althafpt**
created doci=umentation for qm monitor

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qmmonitor.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# qm monitor<vmid>
+
+> To Enter a Qemu Monitor Interface.
+> Refer for more information <https://pve.proxmox.com/pve-docs/qm.1.html>. 
+
+<vmid>:Integer(1-N)
+
+-The unique ID of Virtual Machine.
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:8]: I'm not sure what this command does, but we use this format in `tldr-pages`
```suggestion
- Enter to a Qemu monitor interface:

`qm monitor {{vm_id}}`
```

File: pages/linux/qmmonitor.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# qm monitor<vmid>
+
+> To Enter a Qemu Monitor Interface.
+> Refer for more information <https://pve.proxmox.com/pve-docs/qm.1.html>.
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:4]: We use this standard
```suggestion
> More information <https://pve.proxmox.com/pve-docs/qm.1.html>. 
```

File: pages/linux/qmmonitor.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# qm monitor<vmid>
+
+> To Enter a Qemu Monitor Interface.
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:3]:
```suggestion
> Enter to Qemu Monitor Interfaces.
```

File: pages/linux/qmmonitor.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# qm monitor<vmid>
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:1]:
```suggestion
# qm monitor
```

File: pages/linux/qmmonitor.md (hunk 3 above)
Comment (althafpt) [reply pages/linux/qmmonitor.md:3]: Thank you for pointing out my errors.I am new to GitHub and it is very helpful.

Comment (navarroaxel) [reply pages/linux/qmmonitor.md:3]: Remember that you can use this GitHub feature <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>.
It's the easier way to apply suggestions in `tldr`

Comment (althafpt) [reply pages/linux/qmmonitor.md:3]: Thank you very much❤️

File: pages/linux/qmmonitor.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# qm monitor
+
+> Enter  to Qemu Monitor Interfaces.
+> More information:<https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+`qm monitor {{vm_id}}
+
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:7]:
```suggestion
- Enter to a Qemu monitor interface:

`qm monitor {{vm_id}}`
```

File: pages/linux/qmmonitor.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# qm monitor
+
+> Enter  to Qemu Monitor Interfaces.
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:3]:
```suggestion
> Enter to Qemu monitor interfaces.
```

File: pages/linux/qmmonitor.md (hunk 5 above)
Comment (navarroaxel) [reply pages/linux/qmmonitor.md]: You must include the example.

Comment (althafpt) [reply pages/linux/qmmonitor.md]: ok

File: pages/linux/qmmonitor.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# qm monitor
+
+> Enter to Qemu monitor interfaces.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Enter to a Qemu monitor interface:
+
+`qm monitor {{vm_id}}`
+
```
Comment (navarroaxel) [inline pages/linux/qmmonitor.md:9]:
```suggestion
```

**Comment by althafpt** [comment]
Thanks for help😊

File: pages/linux/qmmonitor.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# qm monitor
+
+> Enter to Qemu monitor interfaces.
```
Comment (kbdharun) [inline pages/linux/qmmonitor.md:3]:
```suggestion
> Enter QEMU monitor interfaces.
```

File: pages/linux/qmmonitor.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Enter to a Qemu monitor interface:
```
Comment (kbdharun) [inline pages/linux/qmmonitor.md:6]:
```suggestion
- Enter a QEMU monitor interface:
```

**Review by navarroaxel — APPROVED** [review]

**Comment by althafpt** [comment]
Hello could I know why this was not accepted as a pull request in hacktoberfest

**Pull request merged**